Scripts/download-portal-item-data.swift

```diff
@@ -200,10 +260,19 @@ portalItems.forEach { (portalURLString, portalItems) in
     portalItems.forEach { (portalItem) in
         // Have we already downloaded the item?
         let filename = downloadedItems[portalItem.identifier] ?? portalItem.filename
-        if FileManager.default.fileExists(atPath: destinationURLProvider.makeURL(filename: filename).path) {
+        let tempURL = destinationURLProvider.makeURL(filename: filename)
+        let pathURL = tempURL.pathExtension == "zip" ? tempURL.deletingPathExtension() : tempURL
+        
+        // Check if a single file or a folder exists.
+        var isDirectory = ObjCBool(false)
+        let isFileExist = FileManager.default.fileExists(atPath: pathURL.path, isDirectory: &isDirectory)
+        
+        if isFileExist && isDirectory.boolValue {
+            print("Item \(portalItem.identifier) has already been downloaded, and is extracted to an folder")
```

Comment:
It is easy enough to tell whether the archive item was extracted to a directory by looking inside of `Portal Data`. Could this just be the same message ("Item \(portalItem.identifier) has already been downloaded")? Then the `isDirectory` check wouldn't be needed.

Reply:
Yes, this make sense.